File: Travelo.Model/City.cs
﻿using System;
using System.Collections.Generic;
using System.Text;

namespace Travelo.Model
{
    public partial class City
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string CountryName { get; set; }
        public List<string> Tags { get; set; }
        public byte[] Image { get; set; }

    }
}

File: travelo_mobile/lib/model/city.dart
import 'package:json_annotation/json_annotation.dart';

part 'city.g.dart';

@JsonSerializable()
class City {
  int? id;
  String? name;
  String? countryName;
  List<String>? tags;
  String? image;
  int? numberOfTrips;
  int? lowestTripPrice;

  City();

  factory City.fromJson(Map<String, dynamic> json) => _$CityFromJson(json);
}
